static/_redirects

```diff
@@ -245,6 +247,8 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 /docs/enterprise/reference/manage.html /docs/capabilities/authentication
 /docs/enterprise/reference/reports /docs/capabilities/reports
 /docs/enterprise/external-data /docs/integrations/
+/docs/integrations /docs/capabilities/integrations
+/docs/integrations /docs/capabilities/integrations
```

Comment:
Looks like this is a duplicate of the line above. Was there supposed to be something else here?

Reply:
Thanks, removed.